src/integrationTest/java/uk/gov/hmcts/darts/utils/EventApiStub.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.darts.utils;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.containing;
+import static com.github.tomakehurst.wiremock.client.WireMock.exactly;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.verify;
+
+public class EventApiStub {
+
+    private static final String EVENT_API_PATH = "/events";
+
+    private static final String EVENTS_API_SUCCESS_BODY = """
+          {
+              "code": "200",
+              "message": "OK"
+          }""";
+
+    public void willRespondSuccessfully() {
+        stubFor(post(urlEqualTo(EVENT_API_PATH))
+              .willReturn(aResponse()
+                    .withHeader("Content-Type", "application/json")
+                    .withBody(EVENTS_API_SUCCESS_BODY)));
+    }
+
+    public void verifyDoesntReceiveEvent() {
+        verify(exactly(0), postRequestedFor(urlEqualTo(EVENT_API_PATH)));
+    }
```

Comment:
Consider moving this method to the superclass and adding a `path` param to enable reuse by other stubs. You could then overload this method in the `EventApiStub` class as follows:
```java
    public void verifyDoesntReceiveEvent() {
        verifyDoesntReceiveEvent(EVENT_API_PATH);
    }
```